src/Core/src/Primitives/Keyboard.cs

```diff
@@ -74,11 +76,17 @@ public static Keyboard Url
 		{
 			get { return s_url ??= new UrlKeyboard(); }
 		}
+		
+		/// <include file="../../docs/Microsoft.Maui/Keyboard.xml" path="//Member[@MemberName='None']/Docs/*" />
```

Comment:
You'll also have to add the actual docs for it, not just replace the member name here